tests/helpers/utils/constants.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * Constants - Common constants used across tests
+ * 
+ * This file contains URL patterns, timeouts, and other constants
+ * used throughout the test suite.
+ */
+
+/**
+ * URL patterns for different steps in the application
+ */
+export const URL_PATTERNS = {
+  LANDING_PAGE: /step-1/,
+  SELECT_STATE: /select-state/,
+  DISCLAIMER: /step-2/,
+  LOCATION_INFO: /step-3/,
+  HOUSEHOLD_SIZE: /step-4/,
+  HOUSEHOLD_MEMBER: /step-5/,
+  EXPENSES: /step-6/,
+  ASSETS: /step-7/,
+  PUBLIC_BENEFITS: /step-8/,
+  NEEDS: /step-9/,
+  REFERRAL_SOURCE: /step-10/,
+  ADDITIONAL_INFO: /step-11/,
+  CONFIRM_INFORMATION: /confirm-information/,
+  RESULTS: /results\/benefits/,
+};
+
+/**
+ * White label identifiers
+ */
+export const WHITE_LABELS = {
```

Comment:
Our white label list should be "nc", "co", "co_energy_calculator", and "ma", the others here are referrers I believe.